py2many/clike.py

```diff
@@ -69,12 +69,15 @@
 def class_for_typename(typename, default_type, locals=None) -> Union[str, object]:
     if typename is None:
         return None
+    if typename == "super" or typename.startswith("super()"):
```

Comment:
Why not add `RuntimeError` to the list below?

Reply:
Because catching `RuntimeError` is about as bad as catching `BaseException` IMO.

To be honest, IMO, the exception list here is growing too long, and this `eval` is called way too often on obviously rubbish values.  If we're a bit more selective, the exception list could be shorter.  But this is not a high priority.